docs/notebooks/meta_learners_synthetic_data.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<ul><li>Before the sentence " They approach this problem ... " can we add a reference to a paper/blog or resource? (maybe https://proceedings.mlr.press/v67/gutierrez17a/gutierrez17a.pdf)</li><li>Overall, I will expand a bit in the introduction. It is a bit confusing what "base learners" are from this first paragraph. </li><li>I think is important to comment on the <strong>Unconfoundedness Assumption</strong> (see https://proceedings.mlr.press/v67/gutierrez17a/gutierrez17a.pdf)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>